docs/manual_tests/Workflow.md

```diff
@@ -0,0 +1,90 @@
+# Workflow of App
+
+## Create tasks for a relation and complete the task
+
+Steps
+
+1. Registration/SignUp in app
+1. LogIn in app
+1. Navigating to members section
+1. Sending request to mentor/mentee
+1. Viewing request
+1. Accepting/Rejecting request
+1. Creating task
+1. Task completion
```

Comment:
I know this gets rendered correctly, but it would be nice to have it well numbered in the raw markdown as well.
1. ...
2. ...
(...)